server/views/cms/rest.py

```diff
@@ -0,0 +1,67 @@
+import logging
```

Comment:
I don't think `rest` is a good name... if we want a single file, I'll just go with `cms.py`; otherwise `cms/collections`, `cms/globals`, etc., makes more sense to me.